examples/torchrec_dlrm/README.md

```diff
@@ -0,0 +1,57 @@
+
+### TorchRec DLRM Example
+
+This example shows how to serve a Deep Learning Recommendation Model (DLRM) with TorchRec on a single GPU (CPU is currently not yet supported by this example). It requires at least 15GB of free GPU memory to run.
+The DLRM is an open source model for personalization and recommendation use cases published by Meta. More information can be found in this [paper](https://arxiv.org/abs/1906.00091) and this [blog post](https://ai.facebook.com/blog/dlrm-an-advanced-open-source-deep-learning-recommendation-model/).
+TorchRec is Meta's open source library for recommender systems in Pytorch. More information on TorchRec can be found in the [official docs](https://pytorch.org/torchrec/).
+
+# Install torchrec using the following command
+This needs pytorch 1.12 to work
+
+```bash
+pip install torchrec==0.2.0
+```
+
+
+--- **This example requires CUDA version >= 11.3**  ---
+
+In this example we will first create and archive the DLRM into a mar file which is subsequently registered in a TorchServe instance. Finally, we run inference using curl.
+
+To create the model and archive it we simple need to run
+
```

Comment:
Torchrec inference is  based on grpc. I don't see any implementation to create grpc request to torchrec. Does I miss any other PRs?

Could you please add test log for this example?

Reply:
That may be happening in the official example of  torchrec (torch deploy + dlrm) . Here we are just loading dlrm model in torchserver . Added end-to-end log